app/models/export.rb

```diff
@@ -69,56 +69,26 @@ def filtered?
     procedure_presentation_id.present?
   end
 
-  def flash_message
-    if available?
-      "L’export au format \"#{format}\" est prêt. Vous pouvez le <a href=\"#{file.url}\">télécharger</a>"
-    else
-      "Nous générons cet export. Veuillez revenir dans quelques minutes pour le télécharger."
-    end
-  end
-
   def self.find_or_create_export(format, groupe_instructeurs, time_span_type: time_span_types.fetch(:everything), statut: statuts.fetch(:tous), procedure_presentation: nil, force: false)
     export = create_or_find_export(format, groupe_instructeurs, time_span_type: time_span_type, statut: statut, procedure_presentation: procedure_presentation)
 
     if export.available? && export.old? && force
```

Comment:
```suggestion
    if export.available? && export.old? || force
```

pas de votre faute, mais en testant, je me rends compte qu'on peut uniquement forcer la re-génération d'un export si celui ci `export.available? && export.old? && force` (cas du bouton regenerer l'export qui est un refresh dans l'ui), ne souhaiterions nous pas permettre de re-generer un export qui fait que planter ?

Reply:
En creusant ce sujet et les raisons qui font qu'on peut redemander un export, je m'aperçois que le "problème" est plus profond : 

- le fait de détruire l'export précédent le fait disparaître de la liste des exports ; ça pourrait être perturbant mais à la limite c'est peut-être pas si grave (bien qu'on romp le contrat de disponibilité sous 32h)
- surtout, entre la première génération et la regénération, le contenu de la procédure présentation peut avoir évolué, (ce qui gardera l'ancien export et en crééra un nouveau car la key de la présentation a changé). Ce n'est donc pas vraiment une version mise à jour du même export comme on le laisse supposer
- => 2 comportements possibles différents, en plus d'être parfois perturbants

Historiquement, j'imagine que la fonction de recréation était nécessaire car dans le dropdown on mettait le lien vers l'export créé. C'était le moyen de redemander le même (on mettait le lien car la présentation était la même !)

Maintenant j'ai l'impression que ce besoin  de génération s'estompe fortement : le dropdown a maintenant tjs un lien "statique" vers /download_exports qui soit lance la génération de l'export, soit ressert l'existant s'il n'est pas stale

Je propose de retirer cette fonctionnalité de retry, sauf pour les cas d'erreur pour lesquels elle est indispensable (bien que là aussi le procedure presentation a pu changer !)

Plus tard si on voulait vraiment re-proposer cette fonctionnalité, on pourrait faire évoluer le modèle Export, en y copiant le contenu de la présentation pour réutiliser les mêmes critères

Qu'en pensez-vous ?